workers/data_refinery_workers/processors/test_array_express.py

```diff
@@ -127,7 +127,7 @@ def test_failure(self):
                        "input_file": input_file}
 
         job_context = array_express._determine_brainarray_package(job_context)
-        failure_reason_template = """unable to read Affy header in input file /home/user/data_store/temp/TEST2/AFFY_TO_PCL/batch_{0}/dummy due to error: Error in (function (filename, info = c("basic", "full"), verbose = FALSE)  : \n  Is /home/user/data_store/temp/TEST2/AFFY_TO_PCL/batch_{1}/dummy really a CEL file? tried reading as text, gzipped text, binary, gzipped binary, command console and gzipped command console formats\n"""  # noqa
+        failure_reason_template = """Unable to read Affy header in input file /home/user/data_store/temp/TEST2/AFFY_TO_PCL/batch_{0}/dummy while running AFFY_TO_PCL due to error: Error in (function (filename, info = c("basic", "full"), verbose = FALSE)  : \n  Is /home/user/data_store/temp/TEST2/AFFY_TO_PCL/batch_{1}/dummy really a CEL file? tried reading as text, gzipped text, binary, gzipped binary, command console and gzipped command console formats\n"""  # noqa
```

Comment:
Does this string, and the one on line 204 need to be all in the same line or can they be broken up to fit our 80-char limit?